runtime/autoload/msgpack.vim

```diff
@@ -62,10 +63,10 @@ function s:msgpack_init_python() abort
               \. "  timestamp = datetime.datetime.strptime(timestr, fmt)\n"
               \. "  timestamp = int(timestamp.timestamp())\n"
               \. "  if timestamp > 2 ** 31:\n"
-              \. "    tsabs = abs(timestamp)"
+              \. "    tsabs = abs(timestamp)\n"
               \. "    return ('{\"_TYPE\": v:msgpack_types.integer,'\n"
               \. "            + '\"_VAL\": [{sign},{v1},{v2},{v3}]}').format(\n"
-              \. "              sign=1 if timestamp >= 0 else -1,\n"
+              \. "              sign=1 if timestamp >= 0 else sign=-1,\n"
```

Comment:
@ZyX-I I took a guess at how to fix this, can you help? This is the failing test:

```
[  ERROR   ] /home/vagrant/neovim/test/functional/plugin/shada_spec.lua @ 2077: autoload/shada.vim function shada#get_binstrings works
stack traceback:
        test/functional/helpers.lua:99: in function 'shada#get_binstrings'
        /home/vagrant/neovim/test/functional/plugin/shada_spec.lua:2069: in function 'getbstrings_eq'
        /home/vagrant/neovim/test/functional/plugin/shada_spec.lua:2083: in function </home/vagrant/neovim/test/functional/plugin/shada_spec.lua:2077>
```

Reply:
This is kwarg("sign", ternary(cond=ge(var("timestamp"), num("0")), if_true=num("1"), if_false=num("-1"))), it is Python ternary operator used to determine value of a keyword argument. What you have here now is not going to work, not only assignment is not an expression, *there is no assignment here*. And I do not think this is full error message.

Reply:
@ZyX-I Yes this doesn't work, I did not want to continue debugging it until getting your opinion. Do we even need `s:msgpack_init_python()` anymore?

Reply:
@ZyX-I what did you intend with `timestamp = int(timestamp.timestamp())` ? `timestamp()` function does not exist on that object in python2 nor python3.